runners/google-cloud-dataflow-java/src/main/java/org/apache/beam/runners/dataflow/util/CloudObject.java

```diff
@@ -182,4 +183,18 @@ public String getClassName() {
   public CloudObject clone() {
     return (CloudObject) super.clone();
   }
+
+  @Override
+  public boolean equals(Object otherObject) {
```

Comment:
Overriding equals because spotBugsMain complained as below:
![image](https://user-images.githubusercontent.com/28604/72624942-cb87ea00-3915-11ea-8610-9da7fbc227ff.png)

Google-http-client's GenericData has its own equals method ([googleapis/google-http-java-client#589](https://github.com/googleapis/google-http-java-client/pull/589/files#diff-914cd7ff18143b3d2398149e1cfb4f45R201)) now.